packages/client/src/types.ts

```diff
@@ -36,6 +36,8 @@ export interface WorkflowExecutionInfo {
   taskQueue: string;
   status: { code: proto.temporal.api.enums.v1.WorkflowExecutionStatus; name: WorkflowExecutionStatusName };
   historyLength: number;
+  // Only available in server version 1.20 or later
+  historySizeBytes?: number;
```

Comment:
We have history length and history size in all SDKs. I can understand how it is confusing because of how people use size in other places. But you could turn around and say "historyLength" is confusing because in some places length is meant to be the byte size (such as HTTP response headers and string length in Go). So we should have called it "historyEventCount".

But that's overthinking it.

Reply:
+1 to remove the `Bytes` suffix, for consistency (both in the TS SDK itself, and across SDKs)

Reply:
Done